pom.xml

```diff
@@ -56,10 +56,15 @@
     <properties>
         <oap-logstream.project.version>17.9.0.0</oap-logstream.project.version>
 
-        <oap.deps.oap.version>17.11.0.0</oap.deps.oap.version>
+        <oap.deps.oap.version>17.10.32.10-oap_ds_parquet-1641</oap.deps.oap.version>
         <oap.deps.oap.tsv.version>17.4.1.8</oap.deps.oap.tsv.version>
+        <oap.deps.parquet.version>1.12.3</oap.deps.parquet.version>
```

Comment:
Why `OAP` prefix is present? This is `org.apache.parquet:parquet-hadoop-bundle`
Same for below libraries

Reply:
These are dependencies for oap